stable/datadog/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## 2.0
 
+## 2.0.12
+
+* Mount the directory containing the CRI socket instead of the socket itself
+  This is to handle the cases where the docker daemon is restarted.
+  In this case, the docker daemon will recreate its docker socket and,
+  if the container bind-mounted directly the socket, the container would
+  still have access to the old socket instead of the one of the new docker
+  daemon.
+  ⚠ This version of the chart requires an agent image 7.19.0 or more recent
```

Comment:
Use https://github.com/Masterminds/sprig/blob/master/docs/semver.md to add an enforcement check in the chart.

Reply:
I think we need to bump to `2.3.0` for this kind of change.

Reply:
I’ve bumped the version and added a check for the agent image version.
The drawback of the agent image version check is that if the tag isn’t a semantic version, it will abort.